resources/js/src/app/components/itemList/filter/ItemColorTiles.js

```diff
@@ -1,6 +1,5 @@
 import url from '../../../mixins/url'
 import { SVGInjector } from '@tanem/svg-injector'
-import DOMPurify from 'dompurify'
```

Comment:
Seems like the compiler didn't care after all, and didn't even include this in the first place. In that case no customer was affected by this anyway. Although you already indicated this by click-testing, the fact that the bundle itself wasn't changed confirmed that.